flixel/FlxObject.hx

```diff
@@ -946,6 +947,47 @@ class FlxObject extends FlxBasic
 	}
 	
 	/**
+	 * Aligns this FlxObject to a target by an anchor point.
+	 * 
+	 * @param	target	To target object to be aligned with.
+	 * @param	anchor	On what anchor point to align the object - default is CENTER.
+	 * @return 	This FlxObject for chaining
+	 */
+	public function alignTo(target:FlxObject, ?anchor:FlxAlign):FlxObject
+	{
+		if (anchor == null)
```

Comment:
NOTE: I would prefer not to take a default parameter (otherwise the function should be called CenterOn), and I agree we this function should take in an optional offset.

I don't want to add this to the FlxObject class, seems like we could easily encapsulate all behavior inside the new FlxAlign class (via [static extension](http://haxe.org/manual/lf-static-extension.html)). That way we avoid increasing the memory footprint of every FlxObject, and this logic becomes totally incapsulated in its own FlxAlign class.

Using a [static extension](http://haxe.org/manual/lf-static-extension.html), you would still be able to do the following without bloating FlxObject:
```
import flixel.util.FlxAlign;
var flxObj = new FlxObject();
var target = new FlxObject();
flxObj.alignTo(target, anchor, offset);
```